src/TouristInfoCards.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
Can you think of creating some functionalities and not using just html to display cards for each city? this makes your code more efficient and nicer